packages/styles/src/components/form-validation.scss

```diff
@@ -90,6 +90,14 @@
   display: block;
 }
 
+.success {
```

Comment:
Why did you create a new class instead of updating the `.is-valid` styles directly?

Reply:
Good question. I separated both to match select style as in https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/branch/MVANE2KzNGq2et3q6T15PC/Components-Post?node-id=7802%3A33963&mode=dev
It's true that in the input mockup there is no such indication though.

Reply:
Good question. I separated both to match select style as in https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/branch/MVANE2KzNGq2et3q6T15PC/Components-Post?node-id=7802%3A33963&mode=dev
It's true that in the input mockup there is no such indication though.

Reply:
Select styles were updated, there is now only one success state which should probably be handled with the `.is-valid` class.

Reply:
Ok, thanks for your investigation.